torch/distributed/fsdp/_optim_utils.py

```diff
@@ -1269,3 +1289,39 @@ def _optim_state_dict(
         )
 
     return fsdp_osd
+
+
+def _get_fqn_to_fsdp_param_info(model: nn.Module) -> Dict[str, FSDPParamInfo]:
+    """
+    Construct the maaping from a param's fqn to its corresponding FSDPParamInfo
+    if the param is managed by FSDP. FlatParam only stores the first FQN of a
```

Comment:
nit: I got confused at first, but I think I understand:
```suggestion
    if the param is managed by FSDP. FlatParameter._fqns only stores the first FQN of a
```
(add backticks if you want, but maybe not to be consistent with rest of the comment)

Reply:
This is correct, thanks!